models/fvns_mcast_addr_blk.go

```diff
@@ -0,0 +1,91 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ciscoecosystem/aci-go-client/container"
+)
+
+const (
+	DnfvnsMcastAddrBlk        = "uni/infra/maddrns-%s/fromaddr-[%s]-toaddr-[%s]"
+	RnfvnsMcastAddrBlk        = "fromaddr-[%s]-toaddr-[%s]"
+	ParentDnfvnsMcastAddrBlk  = "uni/infra/maddrns-%s"
+	FvnsmcastaddrblkClassName = "fvnsMcastAddrBlk"
+)
+
+type AbstractionofIPAddressBlock struct {
```

Comment:
MulticastAddressBlock

Also change all the AbstractionofIPAddressBlock -> MulticastAddressBlock

Reply:
done